backend/app/repositories/helpOffers/update.ts

```diff
@@ -0,0 +1,26 @@
+import db from "../../db/connection";
+
+import { HelpOffer } from "../../db/seeds/data/types/data.types";
+
+export const update = async (user_id: string, helpOfferBody:any): Promise<HelpOffer> => {
```

Comment:
user_id we re receiving has to be a number, at least it should be converted in the controller